source/common/http/http3_status_tracker.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "envoy/event/dispatcher.h"
+#include "envoy/event/timer.h"
+
+namespace Envoy {
+namespace Http {
+
+// Tracks the status of HTTP/3 being broken for a period of time
+// subject to exponential backoff.
+class Http3StatusTracker {
+public:
+  Http3StatusTracker(Event::Dispatcher& dispatcher);
```

Comment:
`explicit`

Reply:
Done. I'm surprised that tidy didn't catch that.